product_docs/docs/biganimal/release/administering_cluster/projects.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Managing projects
+description: "Use projects to provide additional security and segregation to workflows within your organization"
+---
+
+Projects are a group of database clusters within a BigAnimal organization. Access to projects is controlled by a user's role. With projects, you can separate workflows to provide secure and isolated environments and assign different users to different projects. See [Managing portal access](/biganimal/release/administering_cluster/01_portal_access) for more information.
+
+Projects actions are available from the left navigation bar. The actions apply to the project in focus. You can switch between projects using the Projects menu.
+
+## Adding a user to a project
+Before adding a user to a project: 
+- The user must be added through your organization's identify provider.
+
+- The user must log in to the BigAnimal portal at least once.
+- You must be an organization owner or the project owner of the project.
+
+To add a user: 
+1. Select the project you want to add a user to from the Projects menu.
+
+1. Select Users from the left navigation bar.
+2.  Select the edit icon for the user.
+3.  Select **Assign Roles**.
+4.  Depending on the level of access you want for the user, select the appropriate role.
+
+5.  Select **Submit**.
+
+
+## Creating a project
+
+Before creating a new project: 
+
+- You must create a new Azure subscription or AWS account to create a new project. 
+
+  - To create a new Azure subscription, see [Create a Microsoft Customer Agreement subscription](https://learn.microsoft.com/en-us/azure/cost-management-billing/manage/create-subscription).
+
+  - To create a new AWS account, see [Creating an AWS account in your organization](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_create.html).
+
+  !!!note
+     A single Azure subscription or AWS account can only be used to register one project at a time. For example, if you have two projects and only need to deploy your clusters in AWS, you need to create a different AWS account for each project.
+
+     If you want to reuse a cloud account (an Azure subscription or AWS account), you have to delete the project that was previously connected with that account first.
+  !!!   
+- Have an organization owner role.
+
+To create a new project:
+
+1. On the Projects menu, select **New Project**.
+
+1. Enter a unique name for the project.
+1. Select **Create Project**.
+1. Select **Project > See All Projects**.
+1. Select your new project.
+1. Set up the cloud provider for the project. See [Connecting your cloud](/biganimal/latest/getting_started/02_connecting_to_your_cloud/)
```

Comment:
if it's not mentioned somewhere already, I think we should mention that:

- the user can connect a cloud provider that is different from the one they used to register the project
- if they want to connect a different subscription/account of the same CSP the used to register the project, they would have to create a new project.

For example, if they used an azure subscription to set up the project, they can connect an AWS cloud at this step. But they can't connect a new azure subscription to this project.

Reply:
@amrita-suresh I discussed this with Jason and we feel that we have this covered.